Update PyPI

NicolaiSoeborg opened the pull request.

NicolaiSoeborg committed:
Update PyPI

alex-nitrokey commented:
thanks for your PR. There are multiple issues though.

* the `doc` url is not publicly visible. Do you may have another source?
* `multipleu2f` is enabled but no `u2f`. is u2f supported at all?
* `twitter` handle needs to be removed if `hardware` tokens are supported
* please mind the order of the tags (`multipleu2f` after `u2f` and `hardware`)

NicolaiSoeborg committed:
Update PyPI

NicolaiSoeborg commented:
* Changed the `doc` url to a public url
 * Added the `u2f` field (don't know why I forgot that in the first place)
 * Removed `twitter` tag
 * Fixed order of tags (I can't find if any specific order should be enforced? But `multipleu2f` is now in the end)

alex-nitrokey committed:
Update doc link

alex-nitrokey commented:
Thanks a lot for the improved PR, looks good! The doc url changed again apparently, I changed it accordingly.
Will merge as soon travis checks are ready.
